model/model.go

```diff
@@ -4,44 +4,46 @@ import (
 	"sync"
 )
 
+// CallBack represents a callback object with the necessary fields
```

Comment:
Need to mention what the fields are for.

Reply:
done